lib/stanwood/project_creator.rb

```diff
@@ -15,5 +23,19 @@ def execute(project_name, platform)
         exit
       end
     end
+
+    def executeTemplates(platform)
+      case
+      when platform == "--ios"
+        system("curl \"https://raw.githubusercontent.com/stanwood/Xcode_Templates_iOS/master/Script/install.sh\" | sudo bash")
```

Comment:
if the user has the CLI installed, why does the tool then go and fetch the project template installer script from the network? Shouldn't it make more sense to have that available locally? Plus, I'd be weary that the path 'raw.githubusercontent.com' could change under us. I think that path is just an implementation detail in GitHub, it's not an officially supported way of storing files.